packages/image_picker/image_picker/example/ios/RunnerTests/ImagePickerPluginTests.m

```diff
@@ -23,6 +23,7 @@ - (UIViewController *)presentedViewController {
 
 @interface FLTImagePickerPlugin (Test)
 @property(copy, nonatomic) FlutterResult result;
+@property(copy, nonatomic) NSDictionary *arguments;
```

Comment:
We should absolutely not make this design anti-pattern (see my other comments) something that tests rely on and prevent removing. We should test outcomes (expected calls to mocked system APIs, and/or expected return values based on those mocks), not implementation details.